src/app/campaign-list/campaign-list.module.ts

```diff
@@ -53,9 +53,6 @@ import { CampaignListService } from './campaign-list.service';
   ],
   exports: [
     CampaignListComponent
-  ],
```

Comment:
CampaignListService is now `providedIn: 'root'` as a [singleton](https://angular.io/guide/singleton-services)